### ISSUE TITLE
(5.0.0) Some properties are not parsed properly

### ISSUE THREAD

**Issue by leopragi** [Opteo/google-ads-api#224]
Query:
```
SELECT 
    campaign.name,
    campaign.resource_name,
    campaign.status,
    campaign.id,
    campaign.start_date,
    campaign.end_date,
    campaign_budget.name,
    campaign_budget.status,
    campaign_budget.amount_micros
FROM 
    campaign
```

Response: 
```
campaign: { ... },
campaign_budget: {
    name: 'xxxxxxxxx',
    status: 1,
    amount_microscampaign: undefined,
    resource_name: 'xxxxxxxxx/xxxxxxxxx/xxxxxxxxx/xxxxxxxxx'
}
```

Note `amount_microscampaign` should be `amount_micros` inside `campaign_budget`.

**Comment by WillCooter** [comment]
Well spotted, we forgot to add support for multi-line queries. We'll get on this asap

**Comment by WillCooter** [comment]
Fixed in `v5.0.2`
